static/js/containers/CourseSearchPage.js

```diff
@@ -216,7 +213,12 @@ export class CourseSearchPage extends React.Component<Props, State> {
 
   render() {
     const { match, facets, total, loaded, processing } = this.props
-    const { text, error, topics, platforms, availabilities } = this.state
+    const { text, error, activeFacets } = this.state
+    const facetDisplayMap = [
+      ["topics", "Subject Area", null],
+      ["availability", "Availability", _.capitalize],
+      ["platform", "Platform", _.upperCase]
+    ]
```

Comment:
unless this object is going to be mutated at runtime it would be good to pull it out as a separate constant (so we don't redeclare it on every render call)

Reply:
✔️